recipes/basemap-data-hires/bld.bat

```diff
@@ -0,0 +1,10 @@
+if "%PY_VER%" == "2.7" (
+    echo "OK"
+) else (
+    mkdir %LIBRARY_PREFIX%\share\basemap  || exit 1
+)
```

Comment:
Much better! (I had to dig my brain back to 1995 to come up with that horrible if-clause above.)